modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/registry/DDMServiceUpgradeStepRegistrator.java

```diff
@@ -663,6 +665,11 @@ protected String[][] getTableAndPrimaryKeyColumnNames() {
 				}
 
 			});
+
+		registry.register(
+			"5.7.0", "5.7.1",
+			UpgradeProcessFactory.alterColumnType(
+				"DDMTemplate", "name", "TEXT null"));
```

Comment:
@pedro-oliveira446 please check if it's necessary/possible to be a macro change

Reply:
https://liferay.atlassian.net/wiki/spaces/ENGCOREUPGRADE/pages/1654719711/Upgrade+Development+Training#Major-changes